code/modules/hallucinations/types/basic.dm

```diff
@@ -217,7 +220,7 @@
 					to_chat(holder, SPAN_DANGER("You feel a throbbing pain in your [O.name]!"))
 				if(HAL_POWER_MED to INFINITY)
 					to_chat(holder, SPAN_DANGER("You feel an excruciating pain in your [O.name]!"))
-					holder.emote("me",1,"winces.")
+					holder.emote("whimper")
```

Comment:
just as a note you can make these audible or visible by doing `emote("walter", AUDIBLE_MESSAGE or VISIBLE_MESSAGE)`